ui/pages/Project/components/FamilyTable/IndividualRow.jsx

```diff
@@ -171,11 +172,25 @@ const DataDetails = React.memo(({ loadedSamples, individual, mmeSubmission }) =>
         />
       ) : <MmeStatusLabel title="Submitted to MME" dateField="lastModifiedDate" color="violet" individual={individual} mmeSubmission={mmeSubmission} />
     )}
-    <ShowRnaSeqOutliers
-      familyGuid={individual.familyGuid}
-      sample={loadedSamples.find(({ sampleType, isActive }) => isActive && sampleType === SAMPLE_TYPE_RNA)}
-      hasRnaOutlierData={individual.hasRnaOutlierData}
-    />
+    {SHOW_DATA_MODAL_CONFIG.filter(({ shouldShowField }) => individual[shouldShowField]).map(
+      ({ modalName, title, linkText, component }) => {
+        const sample = loadedSamples.find(({ sampleType, isActive }) => isActive && sampleType === SAMPLE_TYPE_RNA)
+        const titleIds = { sampleId: sample.sampleId, individualId: individual.individualId }
```

Comment:
what if there is no matched `sample`? Then `sample.sampleId` will error, so we need to do `sample?.sampleId``